chore: update http error codes (#27775)
supabase/supabase: apps/docs/content/guides/platform/http-status-codes.mdx

4XX status codes indicate an issue on the client's end with the request being made. These could include missing or invalid auth information, a malformed request, making too many requests in too short a time period ("rate limiting"), or a network issue on the client's end.

### 402 service restriction

> In case the Fair Use Policy is applied, the projects of your organization may be restricted from processing requests. If service restrictions are applied, projects will return a 402 status code together with a description on why the project is restricted. The description can contain one or more of the following codes:

Reviewer: ```suggestion
In case the Fair Use Policy is applied, the projects of your organization may be restricted from processing requests. If service restrictions are applied, projects will return a 402 status code together with a description of why the project is restricted. The description can contain one or more of the following codes:
```

> - `exceeded_*` indicates that the project has continued to exceeded the usage limits of their quota, e.g. `exceed_egress_quota`, `exceed_db_size_quota`.

Reviewer: ```suggestion
- `exceeded_*` indicates that the project has continued to exceeded the usage limits of its quota, e.g. `exceed_egress_quota`, `exceed_db_size_quota`.
```

- `overdue_payment` indicates that the organization has overdue bills.

> You will still have access to your data through the Supabase dashboard when Fair Use Policy is applied.

Reviewer: ```suggestion
You will still have access to your data through the Supabase dashboard when the Fair Use Policy is applied.
```

> Head to the [Fair Use Policy](../platform/billing-faq#fair-use-policy) section to learn more about the restrictions and how to remove them.

Reviewer: ```suggestion
See the [Fair Use Policy](/docs/guides/platform/billing-faq#fair-use-policy) section to learn more about the restrictions and how to remove them.
```
